dymos/transcriptions/transcription_base.py

```diff
@@ -170,6 +170,14 @@ def setup_controls(self, phase):
                                 subsys=control_group)
 
             for name, options in phase.control_options.items():
+                if np.any(np.asarray(self.grid_data.subset_num_nodes_per_segment['control_disc']) <= 3) \
```

Comment:
There's nothing wrong with this as is, and I should have caught this previously, but we should move this check into the `phase._check_control_options` method that is called on line 162.  This will keep all of our "sanity checking" code in a single place.